ddp-workspace/projects/ddp-sdk/src/lib/components/login/signInOut.component.ts

```diff
@@ -27,6 +28,8 @@ export class SignInOutComponent implements OnInit, OnDestroy {
     @Input() signInCaption: string;
     @Input() signOutCaption: string;
     @Input() isScrolled: boolean;
+    @Output() signIn = new EventEmitter<void>();
+    @Output() signOut = new EventEmitter<void>();
```

Comment:
Just noticed that it is never used.

Reply:
Right. signIn is used, but signOut is not. I think that is OK. I like the symmetry in/out on/off, etc.